.github/workflows/elixir-build-and-test.yml

```diff
@@ -17,22 +17,22 @@ jobs:
       PGPASSWORD: postgres
     strategy:
       matrix:
-        elixir: ["1.11.4", "1.12.3", "1.13.4", "1.14.4", "1.15.5"]
-        otp: ["22.3", "23.3.4", "24.3.4", "25.3.2", "26.0.2"]
+        elixir: ["1.11.4", "1.12.3", "1.13.4", "1.14.4", "1.15.5", "1.16.2"]
+        otp: ["22.3", "23.3.4", "24.3.4", "25.3.2", "26.1.2"]
```

Comment:
Instead of all the exclusions below it might be helpful to move to a list of versions that we check
```
matrix:
  include:
    - elixir-version: '1.12.1'
      otp-version: '24.0'
    - elixir-version: '1.11.4'
      otp-version: '23.3'
    - elixir-version: '1.10.4'
      otp-version: '22.3'
```

Reply:
Moved to explicitly specifying versions in 31c27580b3d10ec40cbca65c10b98a7256197346 👍